tmt/_compat/typing.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import annotations
+
+import sys
+
+if sys.version_info >= (3, 10):
```

Comment:
Note that there is https://docs.python.org/3/library/platform.html#platform.python_version (not saying one is better than the other)

Reply:
Oh, didn't know about that. Quite subtle in the type used for comparison `int` vs `str`
https://note.nkmk.me/en/python-sys-platform-version-info/

Reply:
below is [python_version_tuple](https://docs.python.org/3/library/platform.html#platform.python_version_tuple) :)

Reply:
Yeah, that's what I was referring to, that one gives a tuple of strings, vs `sys.version_info` which can compare tuple of ints

Reply:
I see, sorry for the confusion. sys makes more sense here then. Thanks!